logger/tests/telemetry.rs

```diff
@@ -2,12 +2,19 @@
 
 use std::time::Duration;
 
-use iroha_logger::{info, init, Configuration, Telemetry, TelemetryFields};
+use iroha_config::base::proxy::Builder;
+use iroha_logger::{info, init, ConfigurationProxy, Telemetry, TelemetryFields};
 use tokio::time;
 
 #[tokio::test]
 async fn telemetry_separation_default() {
-    let (mut receiver, _) = init(&Configuration::default()).unwrap().unwrap();
+    let (mut receiver, _) = init(
+        &ConfigurationProxy::default()
+            .build()
+            .expect("Default logger should always build"),
```

Comment:
```suggestion
            .unwrap(),
```

Reply:
So the difference between two messages signifies that one is an integration test, and the other a unit one, I see